PostalRateCalculator-Frontend/src/components/Test.vue

```diff
@@ -0,0 +1,73 @@
+<template>
```

Comment:
what is Test.vue for?

Reply:
that wa me testing different layouts. I can't safely delete it from my branch i have to wait till it's on master